video_xblock/tests/test_fields.py

```diff
@@ -0,0 +1,78 @@
+"""Tests for classes defined in fields.py."""
+
+import datetime
+import unittest
+
+from video_xblock.fields import RelativeTime
+
+
+class RelativeTimeTest(unittest.TestCase):
+    """
+    Test RelativeTime field.
+    """
+    delta = RelativeTime()
+
+    def test_from_json(self):
```

Comment:
https://github.com/raccoongang/xblock-video/blob/tests/backends_tests/video_xblock/tests/test_fields.py
Hm... Docstrings are added. Why don't they visible in PR?